LetSwift/View/Events/Views/EventsView.swift

```diff
@@ -71,10 +76,12 @@ fileprivate extension EventsView {
     func companyEvents() -> some View {
         LazyVGrid(columns: gridLayout, spacing: 0) {
             ForEach(Array(StampImagePosition.positions.enumerated()), id: \.offset) { (index, position) in
+                
                 eventCell(at: index)
                     .overlay {
-                        if index < eventCellStates.count {
-                            Image(uiImage: eventCellStates[index].image)
+                        if index < viewModel.eventCellStates.count {
```

Comment:
```suggestion
                        if viewModel.eventCellStates.indices.contains(index) {
```

viewModel에서 가지고 있는 model을 조회 하는 용도면 `contains` 사용하는게 더 직관적일 것 같아요-! 검토 부탁드려요~

Reply:
> viewModel에서 가지고 있는 model을 조회 하는 용도면 `contains` 사용하는게 더 직관적일 것 같아요-! 검토 부탁드려요~

698cb24
index를 사용하는 코드 맥락을 일치시킨다는 점에서 말씀하신 방향이 더 좋은 것 같습니다! 수정완료했습니다.